webapp/src/app/api/v0/cas/config.service.ts

```diff
@@ -0,0 +1,147 @@
+import crypto from "crypto";
+import { List, Map } from "immutable";
+import toml from "toml";
+
+import { Injectable } from "@angular/core";
+
+import { InstanceID } from "@dedis/cothority/byzcoin";
+
+export class Action {
```

Comment:
I'd take a small comment here (and other exported symbols) for two kind of persons:
- you in 2 months
- everybody else